run/shared/species_database.yml

```diff
@@ -1302,37 +1302,180 @@ DMS:
   Henry_K0: 0.48
   Is_Aerosol: true
   MW_g: 62.13
-DST1:
-  << : *DST1properties
-  FullName: Dust aerosol, Reff = 0.7 microns
+DSTbin1_PROP: &DSTbin1properties
```

Comment:
Is it possible to have a DST_PROP that can be used for all of the DSTbin*_PROP, just to reduce the list for the individual bin properties entries? It seems like there is a lot of overlap across the properties and would be nice to list just the values that are different.

Reply:
@lizziel: We now just have a `&DSTbinproperties` with most of the properties for the DSTbin*, DSTALbin*, SO4Dbin*, and NITDbin* species.   That reduces the amount of YAML anchors that we need.